forms/available/edit_type.yaml

```diff
@@ -164,69 +164,77 @@ i18n:
     modification-description:
   it:
     notes-label: Commenti
-    semantic-labels-label: Scegli un significato semantico
+    semanticlabels-label: Scegli l'intenzione più probabile per questa modifica.
     # Semantic meanings
-    link-disambig-label: Collegamento disambiguazione
-    link-disambig-description:
-    fact-update-label: Fatto Aggiornamento
-    fact-update-description:
-    formatting-label: Formattazione
-    formatting-description:
+    copyediting-label: Copy Editing
```

Comment:
It's commonly used in Italian but let me get a second opinion before closing this. 

> On Feb 17, 2016, at 06:19, Aaron Halfaker <notifications@github.com> wrote:
> 
> In forms/available/edit_type.yaml:
> 
> >      # Semantic meanings
> > -    link-disambig-label: Collegamento disambiguazione
> > -    link-disambig-description:
> > -    fact-update-label: Fatto Aggiornamento
> > -    fact-update-description:
> > -    formatting-label: Formattazione
> > -    formatting-description:
> > +    copyediting-label: Copy Editing
> Still English. Is that OK?
> 
> —
> Reply to this email directly or view it on GitHub.
>